vars/commonPipelineEnvironment.groovy

```diff
@@ -49,7 +49,7 @@ class commonPipelineEnvironment implements Serializable {
 
     private Map valueMap = [:]
 
-    void setValue(String property, value) {
+    void setValue(String property, def value) {
```

Comment:
[question] it is better to have the `def`, no doubt about that, but is that required in order to fix the issue?

Reply:
No, just to be safe. I guess I was confused by the way it's handled in GO.